kit/Kit.cpp

```diff
@@ -2343,11 +2376,16 @@ class KitSocketPoll final : public SocketPoll
             if (remainingTime < std::chrono::microseconds(timeoutMicroS))
                 _document->checkIdle();
             else
-                LOG_TRC("Poll of woudl not close gap - continuing");
+                LOG_TRC("Poll of would not close gap - continuing");
         }
 
         drainQueue();
 
+        if (_document)
+        {
+            _document->trimIfExcessive();
```

Comment:
Naming-wise is there anything Excessive here ? =) we have trimIfInactive already trimOnShortIdle ? ;-) up to you but - what was Excessive ?